add-on/chrome/bootstrap.js

```diff
@@ -534,13 +534,13 @@ var WindowListener = {
 
         let box = gBrowser.getNotificationBox();
         let pauseButtonLabel = this._getString(this._browserSharePaused ?
-                                               "infobar_button_resume_label" :
-                                               "infobar_button_pause_label");
+                                               "infobar_button_restart_label2" :
+                                               "infobar_button_stop_label2");
         let pauseButtonAccessKey = this._getString(this._browserSharePaused ?
-                                                   "infobar_button_resume_accesskey" :
-                                                   "infobar_button_pause_accesskey");
+                                                   "infobar_button_restart_accesskey" :
+                                                   "infobar_button_stop_accesskey");
         let barLabel = this._getString(this._browserSharePaused ?
-                                       "infobar_screenshare_paused_browser_message" :
+                                       "infobar_screenshare_stop_sharing_message" :
                                        "infobar_screenshare_browser_message2");
```

Comment:
This duplication seems a bit silly. Perhaps a simple helper can clean it up.

```
let getPausedUnpausedStrings = () => this._browserSharePaused ? {
  label: this._getString("infobar_button_restart_label2"),
  accesskey: this._getString("infobar_button_restart_accesskey"),
  message: this._getString("infobar_screenshare_stop_sharing_message")
} : {
  ...
};
```

Then use it outside and inside the callback.